buildstockbatch/cloud/docker_base.py

```diff
@@ -461,3 +465,53 @@ def run_simulations(cls, cfg, job_id, jobs_d, sim_dir, fs, output_path):
                 shutil.rmtree(item)
             elif os.path.isfile(item):
                 os.remove(item)
+
+    def log_summary(self):
+        """
+        Log a summary of how many simulations succeeded, failed, or ended with other statuses.
+        Uses the `completed_status` column of the files in results_csvs/results_*.csv.gz.
+        """
+        fs = self.get_fs()
+        # Summary of simulation statuses across all upgrades
+        status_summary = {}
+        total_counts = collections.defaultdict(int)
+
+        results_csv_dir = f"{self.results_dir}/results_csvs/"
+        try:
+            results_files = fs.ls(results_csv_dir)
+        except FileNotFoundError:
+            logger.info(f"No results CSV files found at {results_csv_dir}")
+            return
+
+        for result in results_files:
+            upgrade_id = result.split(".")[0][-2:]
+            with fs.open(result) as f:
+                with gzip.open(f) as gf:
+                    df = pd.read_csv(gf, usecols=["completed_status"])
```

Comment:
The parquet files have the same data and are very fast and efficient to read, especially if you're only looking at a few columns.

Reply:
Good point - updated.